src/compose/SubmitButton.js

```diff
@@ -21,16 +21,17 @@ const styles = StyleSheet.create({
   },
 });
 
-export default class SendButton extends React.Component {
+export default class SubmitButton extends React.Component {
```

Comment:
Yeah, Send Button suggest's its being only used to send the message, but submit suggest's it's being used to complete a process (send or edit)